locales/it/index.yml

```
@@ -1240,6 +1240,9 @@ genericError: Si è verificato un errore
 titleUpdateApp: Aggiorna IO!
 msgErrorUpdateApp: "Si è verificato un errore durante l'apertura dello store delle app"
 messageUpdateApp: "IO introduce spesso piccole migliorie e nuove funzioni: per continuare ad usare l'app è necessario aggiornarla all'ultima versione."
+titleUpdateAppAlert: Per continuare, aggiorna IO
+messageUpdateAppAlert: È disponibile una nuova versione di IO su {{storeName}}, scaricala subito per utilizzare questa funzionalità.
```

```suggestion
messageUpdateAppAlert: Per utilizzare questa funzionalità è necessario aggiornare l'app all'ultima versione. Scarica l'aggiornamento da {{storeName}}
```
Btw I'm not sure my suggested sentence is better.
@thisisjp What do you think?

---

I just took it from the pivotal [story comments](https://www.pivotaltracker.com/story/show/175285929/comments/218844818) from @thisisjp 😄